pyproject.toml

```diff
@@ -20,6 +20,9 @@ dependencies = [
   "joblib",
   "pyyaml",
   "scikit-learn",
+  "pytorch-ignite",
+  "gdown",
```

Comment:
why do we need gdown here? Is it for the examples? If so, it should be specified in there and not be installed in Leakpro.